src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1318 @@
+            }
+            $aOutputHeaders[$sSection] = array();
+        }
+
+        // Loop over input headers and link them to output headers, such that
+        // $aOutputHeaders[section][i] = outHeader, where section is the output
+        // section as defined in $aImportSections, i is the index of the input
+        // header and outHeader is the name of the column in the output.
+        for ($i = 0; $i < count($aMatches[1]); $i++) {
+
+            $aSectionIDs = array_keys($aSections);
+
+            // Check if field is manually linked in $aFieldLinks.
+            $sHeader = $aMatches[1][$i];
+
+            // Special consideration for Variant/DNA_published, as it can be linked to two
+            // fields: VariantOnTranscript/Published_as and VariantOnGenome/Published_as,
+            // but the former is preferred.
+            if ($sHeader == 'Variant/DNA_published') {
+                if (!in_array('VariantOnTranscript/Published_as', $aSections['vot']['db_fields']) &&
+                    in_array('VariantOnGenome/Published_as', $aSections['vog']['db_fields'])) {
+                    // Field available on VOG and not on VOT.
+                    $aOutputHeaders['vog'][$i] = 'VariantOnGenome/Published_as';
+                    continue;
+                } else {
+                    // By default put the published_as field on VOT.
+                    $aOutputHeaders['vot'][$i] = 'VariantOnTranscript/Published_as';
+                    continue;
+                }
+            }
+
+            if (isset($aFieldLinks[$sHeader])) {
+                // Use output header linked in $aFieldLinks.
+                list($sSection, $sHeaderOut) = $aFieldLinks[$sHeader];
+                $aOutputHeaders[$sSection][$i] = $sHeaderOut;
+                continue;
+            }
+
+            // Check if header occurs as a literal DB field.
+            foreach ($aSectionIDs as $sSection) {
+                if (isset($aSections[$sSection]['db_fields'])  &&
+                    in_array($sHeader, $aSections[$sSection]['db_fields'])) {
+                    $aOutputHeaders[$sSection][$i] = $sHeader;
+                    continue 2;
+                }
+            }
+
+            // Try to link custom columns.
+            if (strpos($sHeader, '/') !== false) {
+                list(, $sFieldname) = explode('/', $sHeader, 2);
+                foreach ($aSectionIDs as $sSection) {
+                    $aSection = $aSections[$sSection];
+                    if (isset($aSection['customcol_prefix'])) {
+                        $sNewFieldName = $aSection['customcol_prefix'] . '/' . $sFieldname;
+                        if (in_array($sNewFieldName, $aSection['db_fields'])) {
+                            // Set output header with new LOVD3 prefix (e.g. Individual).
+                            $aOutputHeaders[$sSection][$i] = $sNewFieldName;
+                            continue 2;
+                        }
```

Comment:
In this block, you make a new field name for each column with *every* section. Shouldn't you be able to see which section this column belongs to already, by looking at the original prefix?